spec/lib/puppet_spec/compiler.rb

```diff
@@ -7,8 +7,8 @@ def compile_to_catalog(string, node = Puppet::Node.new('foonode'))
     Puppet::Parser::Compiler.compile(node).filter { |r| r.virtual? }
   end
 
-  def compile_to_ral(manifest)
-    catalog = compile_to_catalog(manifest)
+  def compile_to_ral(manifest, node = Puppet::Node.new('foonode'))
```

Comment:
Yes. This is the name of the default node for compile_to_catalog (see lines above the diff).